backend/tests/test_storage.py

```diff
@@ -32,3 +45,76 @@ def test_create_all_tables(raw_connection):
         storage.create_all_tables()
     after = raw_connection.execute(_LIST_TABLES_SQL)
     assert len(list(after)) == 3
+
+
+def test_search_clause__produces_valid_query():
+    """
+    Does the ``search_clause()`` function properly add search to a sql query?
+    """
+    query = config_table.select().where(
+        storage.search_clause(
+            search_terms="foo",
+            searchable_fields=[
+                config_table.c.name,
+                config_table.c.product,
+            ],
+        )
+    )
+
+    rendered_query = storage.render_sql(query)
+    columns_str = ",".join([str(c) for c in config_table.columns])
+    assert query_stripper(rendered_query) == query_stripper(
+        f"""
+        select {columns_str}
+        from config
+        where
+            lower(config.name) like lower('%foo%')
+            or lower(config.product) like lower('%foo%')
+        """
+    )
+
+
+def test_sort_clause__produces_valid_query():
+    """
+    Does the ``sort_clause()`` function properly add sort to a sql query?
+    """
+
+    query = config_table.select().order_by(
+        storage.sort_clause(
+            sort_field="name",
+            sortable_fields=[
+                config_table.c.name,
+                config_table.c.product,
+            ],
+            sort_ascending=False,
+        )
+    )
+
+    rendered_query = storage.render_sql(query)
+    columns_str = ",".join([str(c) for c in config_table.columns])
+    assert query_stripper(rendered_query) == query_stripper(
+        f"""
+        select {columns_str}
+        from config
+        order by
+            config.name desc
+        """
+    )
+
+
+def test_sort_clause__produces_valid_query():
```

Comment:
This test has the same name as the test above.

Reply:
Fixed in commit 18d16f2